mff_rams_plugin/config.py

```diff
@@ -8,3 +8,15 @@ class ExtraConfig:
     @property
     def DEALER_BADGE_PRICE(self):
         return self.get_attendee_price()
+
+    @property
+    def TABLE_OPTS(self):
+      return [ (1.0, 'Single Table'),(2.0, 'Double Table'),(3.0, 'Triple Table'),(4.0, 'Quad Table'),(5.0, '10x10 Booth')]
+
+    @property
+    def ADMIN_TABLE_OPTS(self):
+      return [(0.0, 'No Table')] + TABLE_OPTS()
+   
+    @property
+    def PREREG_TABLE_OPTS(self):
+      return [(count, '{}: ${}'.format(desc, TABLE_PRICES()[count])) for count, desc in TABLE_OPTS()]
```

Comment:
Interesting! That's how they were originally, of course. Property refs vs. function calls still confuse me. I guess....... @property makes it available as some sort of property kinda like it was in development.ini, as opposed to as a fn call?